src/service/poc_mobile.proto

```diff
@@ -7,6 +7,81 @@ import "mapper.proto";
 import "hex_boosting.proto";
 import "service_provider.proto";
 
+message hex_usage_counts_req_v1 {
```

Comment:
Is the `counts` proto name conforming to an existing convention? I could imagine this payload growing to include usage details that are not strictly counts. Total nit, but names like `details` or `stats` may be more generic.  

Same applies to the radio counts message.

Reply:
addressed here: https://github.com/helium/proto/pull/428/commits/1b9d9548a6f7c4d390cf0585b2d7c61f564e0d8a